google/cloud/storage/testbench/testbench.py

```diff
@@ -1251,23 +1251,14 @@ def test_iam_permissions(self, request):
         return result
 
 
-# Define the collection of GcsObjects indexed by <bucket_name>/o/<object_name>
-GCS_OBJECTS = dict()
-
-# Define the collection of Buckets indexed by <bucket_name>
-GCS_BUCKETS = dict()
-
 # Define the WSGI application to handle bucket requests.
 GCS_HANDLER_PATH = '/storage/v1'
 gcs = flask.Flask(__name__)
 gcs.debug = True
 
 
-# TODO(#821) TODO(#820) - until we implement CreateBucket + DeleteBucket insert
-# a well-known bucket based on the BUCKET_NAME environment (this is set by our
-# CI builds), to make the integration tests easy to write.
 def insert_magic_bucket(base_url):
-    if len(GCS_BUCKETS) == 0:
+    if testbench_utils.has_buckets() == 0:
```

Comment:
Change to `if not testbench_utils.has_buckets():`

https://github.com/google/styleguide/blob/gh-pages/pyguide.md#2144-decision

Or, actually... it seems weird to me that there's a similar method named `has_bucket(bucket_name)`, but then `has_buckets()` isn't the plural version of it that accepts a list of buckets to check for.  Might be worth either:
-  making this method return the count of buckets (int) rather than whether the len is 0 (bool), and naming it something like `num_buckets()` or `bucket_count()`
-  or just let callers use `len(testbench_utils.all_buckets())`.  I'd probably opt for this.

Reply:
Done.